cmd/gpu-feature-discovery/main_test.go

```diff
@@ -209,12 +209,12 @@ func TestRunSleep(t *testing.T) {
 				MigStrategy:     ptr("none"),
 				FailOnInitError: ptr(true),
 				GFD: &spec.GFDCommandLineFlags{
-					Oneshot:           ptr(false),
-					OutputFile:        ptr("./gfd-test-loop"),
-					SleepInterval:     ptr(spec.Duration(time.Second)),
-					NoTimestamp:       ptr(false),
-					MachineTypeFile:   ptr(testMachineTypeFile),
-					VfioPciDriverMode: ptr(false),
+					Oneshot:         ptr(false),
+					OutputFile:      ptr("./gfd-test-loop"),
+					SleepInterval:   ptr(spec.Duration(time.Second)),
+					NoTimestamp:     ptr(false),
+					MachineTypeFile: ptr(testMachineTypeFile),
+					Mode:            ptr(""),
```

Comment:
This should ideally default to `auto` -- although we can use `""` to mean this too.

Reply:
Fixed

Reply:
How did you fix this? I would have expected this to change to `ptr("auto")` instead.

Reply:
Apologies I missed this. Fixed it now.